src/apache/conf/ssl.conf

```diff
@@ -88,12 +87,15 @@ SSLRandomSeed connect file:/dev/urandom 512
 
     SSLEngine on
     SSLHonorCipherOrder On
-    SSLCipherSuite ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:ECDH+3DES:DH+3DES:RSA+AESGCM:RSA+AES:RSA+3DES:!aNULL:!MD5:!DSS;
+	SSLCipherSuite ECDHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA256
 
     SSLCertificateFile      ${SNAP_DATA}/certs/live/cert.pem
     SSLCertificateKeyFile   ${SNAP_DATA}/certs/live/privkey.pem
     SSLCertificateChainFile ${SNAP_DATA}/certs/live/chain.pem
 
+    SSLOpenSSLConfCmd Curves brainpoolP512r1:secp521r1:brainpoolP384r1:secp384r1
```

Comment:
For reference, this is how it's supposed to work:
https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslopensslconfcmd

And this is the alternative that we need to use:
https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslcertificatefile